docs/recipes/catalog/ecs-infra-block.yaml

```
@@ -0,0 +1,9 @@
+recipeName: Serverless workflows with ECSTask blocks
```

```suggestion
recipeName: Serverless Workflows with ECSTask Blocks
```
Since title case has become our naming convention.